core/migrations/20221010152624_add_ip_to_user_sessions.php

```diff
@@ -0,0 +1,14 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Migration\AbstractMigration;
+
+final class AddIpToUserSessions extends AbstractMigration
+{
+    public function change(): void
+    {
+        $table = $this->table('nl2_users_session');
+        $table->addColumn('ip', 'string', ['length' => 128, 'null' => true]);
```

Comment:
Would it make sense to have this as a foreign key to the `nl2_users_ips` table instead to save duplicating the IP address in the database?